pymodbus/repl/server/main.py

```diff
@@ -178,29 +178,29 @@ def run(
     data_block_settings = modbus_config.pop("data_block_settings", {})
     modbus_config = modbus_config.get(modbus_server, {})
     modbus_config = process_extra_args(extra_args, modbus_config)
```

Comment:
This is missing in API_CHANGES

Reply:
That specific change does not change the API, it rather corrects the parameter name when instantiating the various Modbus*Server objects. Neither the servers' __init__ methods nor the parameter when calling the console.server was changed.